systemd/schema-registry.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Schema Registry
+Documentation=http://docs.confluent.io/current/schema-registry/docs/index.html
+Requires=network.target
+Wants=kafka-server.service
+After=network.target kafka-server.service
+
+[Service]
+Type=forking
```

Comment:
Does lack of any user info here mean it is running as root? This is fine since we don't have users setup yet, just curious to know exactly how this script will result in it running.